sopel/bot.py

```diff
@@ -594,27 +594,26 @@ def rate_limit_info(
     ) -> tuple[bool, Optional[str]]:
         if trigger.admin or rule.is_unblockable():
             return False, None
-
+        nick = trigger.nick
```

Comment:
```suggestion

        nick = trigger.nick
```

Like that?

Reply:
Yes, like that. I guess it allows you to "put back the deleted line" if you select only the "added" line and edit manually.

All this time and the suggestion UX is still "meh" at best. 🫤